go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/mingfang/alluxio-csi
```

Comment:
can you update this to be `module github.com/Alluxio/alluxio-csi`?

Reply:
done